### PR TITLE
Fixes #14467: AIX agent fails when there is a libcrypto in system libs

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/14467

### CODE REVIEW

File: rudder-agent/SPECS/rudder-agent.spec [hunk 1]
```diff
@@ -62,8 +62,12 @@
 # Default to use PIE code if possible
 %define use_pie true
 
-# Default to build with posix acl support
-%define use_acl true
+# OS family to build for
+%if "%{?_os}" != "aix"
```
Comment (amousset) [inline rudder-agent/SPECS/rudder-agent.spec]: Shouldn't it be the opposite?

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
OK, squash merging this PR

**Comment by peckpeck** [comment]
OK, merging this PR

**Pull request merged**